sdk/examples/how_tos/outputs/output_features.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! In this example we will build NFT outputs with all possible features.
+//!
+//! `cargo run --example output_features --release`
```

Comment:
Can you put `--release` before the example pls?

Reply:
```suggestion
//! `cargo run --release --example output_features`
```